lib/excon/ssl_socket.rb

```diff
@@ -26,7 +28,9 @@ def initialize(data = {})
         ssl_context.ssl_version = @data[:ssl_version]
       end
 
-      if @data[:ssl_verify_peer]
+      env_verify_peer = ENV.has_key?(VERIFY_ENV_VAR) && (ENV[VERIFY_ENV_VAR] && ENV[VERIFY_ENV_VAR] != 'false')
```

Comment:
It took me a couple reads to parse this, though it probably should have been easier for me. Maybe we could do `!([nil,'false'].include?(ENV[VERIFY_ENV_VAR])` to be a little more explicit? What do you think?